### PR TITLE
Add support for projects with just raster files, no annotations

### PR DESCRIPTION
**Pull request by dustymugs**
Depending on the training data, it really isn't feasible to build sufficient negative data for training. Instead, allow configs to pass in rasters to be used as negative data...

```
...
{
    "raster_source": {
        "geotiff_files": {
            "uris": ["s3://aws-naip/ca/2012/1m/rgbir/39122/m_3912232_sw_10_1_20120705.tif"]
        }
    },
    "ground_truth_label_store": {
        "object_detection_geojson_file": {}
    }
}
```
I'm kind of concerned with the verbosity for essentially an empty `ground_truth_label_store` but minimal changes needed to support the end goal...

### CODE REVIEW

**Comment by lewfish** [comment]
In order to actually generate negative chips from projects with no labels I think you'll need to make an additional change here since `len(pos_windows)` will be zero https://github.com/azavea/raster-vision/blob/develop/src/rastervision/ml_tasks/object_detection.py#L68

**Comment by lewfish** [comment]
Otherwise this looks good to me.

**Comment by dustymugs** [comment]
Oops. I'm missing a commit here. Let me go find it from my master branch...

File: src/rastervision/ml_tasks/object_detection.py [hunk 1]
```diff
@@ -65,8 +65,11 @@ def get_train_windows(self, project, options):
         # any labels. It may take many attempts to generate a single
         # negative window, and could get into an infinite loop in some cases,
         # so we cap the number of attempts.
-        nb_neg_windows = \
-            int(options.object_detection_options.neg_ratio * len(pos_windows))
+        if nb_pos_windows:
```
Comment (lewfish) [inline src/rastervision/ml_tasks/object_detection.py]: This needs to be preceded by `nb_pos_windows = len(pos_windows)`

Comment (dustymugs) [reply src/rastervision/ml_tasks/object_detection.py]: Yes... you're correct...

**Comment by lewfish** [comment]
This will be good to go after cleaning up the commits.
